test/utils-tests.jl

```diff
@@ -544,6 +544,20 @@ end
     [-0.37388888888888894, -1.0, -3.0]
 @test ten.([12.0, -0.0], [24, 30]) == ten.([" 12::24", " -0:30: "]) == [12.4, -0.5]
 
+#Test tic_once
+let
```

Comment:
Yes.  I don't recall other global changes I'd like to apply to this package.

But, really, if you enjoy coding and have limited spare time, I suggest you to go ahead with real coding, these are mostly cosmetic, yet useful, changes, you won't learn much by adding a bunch of `@testset`s ;-)